integration-tests/pom.xml

```diff
@@ -0,0 +1,212 @@
+<!-- Copyright 2017, Oracle Corporation. All Rights Reserved. -->
```

Comment:
License template for all files (okay to use XML comment style here)
// Copyright 2018, Oracle Corporation and/or its affiliates.  All rights reserved.
// Licensed under the Universal Permissive License v 1.0 as shown at
// http://oss.oracle.com/licenses/upl.

Reply:
added license in all files